rpmlint/checks/PythonCheck.py

```diff
@@ -0,0 +1,64 @@
+import os
+import re
+
+from rpmlint.checks.AbstractCheck import AbstractFilesCheck
+
+# Warning messages
+WARNS = {'tests': 'python-tests-in-package',
```

Comment:
Actually, I have to correct myself. After some discussion with my colleague, I agree that these should be ERR. Yes, I can see @hroncok ’s [comment](https://github.com/rpm-software-management/rpmlint/pull/430#discussion_r428141147), but I think these packages are truly exceptional and that’s what `rpmlintrc` is for. On the other hand the proliferation of warnings make people just ignore the output of `rpmlint` more and more, it is not healthy. Warning should be just for warnings not for our lack of guts to decide.

Reply:
I don't really care whether this is a warning or an error.

Reply:
I've just pointed out in the comment you link that if it has `__init__.py` it should not produce the warning/error.

Reply:
Except, @danigm , I cannot find the code checking for the presence of `__init__.py`, did I miss it?

Reply:
I don't think it's there either.

Reply:
I've just added a new commit to do the check for `__init__.py` file, so python modules called `doc` won't raise the warning: https://github.com/rpm-software-management/rpmlint/pull/955/commits